mesheryctl/internal/cli/root/mesh/remove.go

```diff
@@ -42,6 +41,11 @@ mesheryctl mesh remove [mesh adapter name]
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
+
+			if len(args) > 0 {
+				return errors.New(utils.MeshError(fmt.Sprintf("'%s' is an invalid argument. 'mesheryctl mesh remove' should not have any arguments.\n", args[0])))
+			}
+
```

Comment:
This command accepts arguments. Example `mesheryctl mesh remove [mesh adapter name]`

Reply:
Ok, `mesheryctl mesh remove` can have 0 or 1 arguments, should I check if `len(agrs) > 1`  or just remove this check here? Previous `Args: cobra.MinimumNArgs(0)` has no meaning as we can't pass negative numbers of argument.

Reply:
Yes, changing it to `len(args) > 1` is better but the error message will also need to be modified.

Reply:
sure we can show message like "'mesheryctl mesh remove' can't handle multiple adapter-name, it can remove only one adapter at a time."